iib/web/iib_static_types.py

```diff
@@ -395,4 +409,17 @@ class BaseClassRequestResponse(APIPartImageBuildRequestResponse, CommonIndexImag
     """Datastructure representing data returned by Request class to_json method."""
 
 
+class FbcOperationRequestResponse(APIPartImageBuildRequestResponse):
```

Comment:
Wondering if we will need attrs from CommonIndexImageResponseBase like internal_index_image or distribution scope

Reply:
we stored those in DB

Reply:
yeah good point, thanks - updated - please check it again